pages/services.js

```diff
@@ -93,6 +95,64 @@ const Services = () => (
           },
         ]}
       />
+      <a
+        name="content-block-cards-and-card-Link"
```

Comment:
Could you please convert this `L` in `content-block-cards-and-card-Link` to lowercase?

Reply:
Done. Thanks @caiobozato